pkg/tsdb/loki/loki.go

```diff
@@ -56,6 +56,7 @@ type ResponseModel struct {
 	LegendFormat string `json:"legendFormat"`
 	Interval     string `json:"interval"`
 	IntervalMS   int    `json:"intervalMS"`
+	Resolution   int64  `json:"resolution"`
```

Comment:
@idafurjes is it correct to specify the type as `int64` here or should it be specified as `int` and then wrapped as `int64`?

Reply:
you can leave the int64, I am maybe not fully understanding what you want to achieve here, maybe you can explain further

Reply:
We use `step := time.Duration(int64(interval.Value) * resolution)` to calculate the step value but `time.Duration()` only accepts `int64`. So we can either wrap resolution in `int64()` or specify it. I'm just wondering if it matters or not or if there's any preference since `IntervalMS` is also an `int` that's being wrapped by `int64()` at line 204.

Reply:
I personally would rather go with int64, you avoid one conversion, if you feel comfortable and think it's cleaner to change the type of IntervalMS to int64, you can do that as well